### DISCUSSION TITLE
How to bootstrap external pex files within running Python interpreter

### DISCUSSION THREAD

**Discussion by dineshkumar20** [pex-tool/pex#2404] [Q&A] [answered]
I'm currently using `pex_bootstrapper.bootstrap_pex_env()` to bootstrap imports from an external PEX file within a running Python interpreter. It's working for the current session, but not persistently across other sessions. The reason is I don't want to bootstrap the PEX file every time because it's taking a few seconds. Is there a way to permanently append the code from the PEX file to `sys.path`?

**Comment by jsirois** [comment, marked as answer]
@dineshkumar20 I missed this yesterday, but it appears to duplicate https://github.com/pex-tool/pex/issues/2405.

Did that issue answer your question or is there more you'd like to discuss?

**Comment by dineshkumar20** [comment]
Thank you so much @jsirois
